packages/ui/src/components/inputs/number-input.tsx

```diff
@@ -113,30 +114,33 @@ export const NumberInput = forwardRef<HTMLInputElement, NumberInputProps>(
           }}
           suffix={
             hideStepper ? null : (
-              <div className="flex flex-col">
-                <Button
-                  tabIndex={-1}
-                  aria-label="Increment value"
-                  variant="ghost"
-                  iconOnly
-                  onClick={handleIncrement}
-                  disabled={disabled}
-                  size="sm"
-                >
-                  <IconV2 name="nav-arrow-up" size={14} />
-                </Button>
-                <hr />
-                <Button
-                  tabIndex={-1}
-                  aria-label="Decrement value"
-                  variant="ghost"
-                  iconOnly
-                  onClick={handleDecrement}
-                  disabled={disabled}
-                  size="sm"
-                >
-                  <IconV2 name="nav-arrow-down" size={14} />
-                </Button>
+              <div className="flex">
+                <div className="flex flex-col">
+                  <Button
+                    tabIndex={-1}
+                    aria-label="Increment value"
+                    variant="ghost"
+                    iconOnly
+                    onClick={handleIncrement}
+                    disabled={disabled}
+                    size="sm"
+                  >
+                    <IconV2 name="nav-arrow-up" size={14} />
+                  </Button>
+                  <hr />
+                  <Button
+                    tabIndex={-1}
+                    aria-label="Decrement value"
+                    variant="ghost"
+                    iconOnly
+                    onClick={handleDecrement}
+                    disabled={disabled}
+                    size="sm"
+                  >
+                    <IconV2 name="nav-arrow-down" size={14} />
+                  </Button>
+                </div>
+                <div className="border-inherit">{suffix}</div>
```

Comment:
Can we render suffix conditionally?

```suggestion
                {suffix && <div className="border-inherit">{suffix}</div>}
```

Reply:
Done